src/Dashboard/components/ButtonDownloadLogs/ButtonDownloadLogs.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Button, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    height: '100%'
+  }
+}));
+
+const ButtonDownloadLogs = (props) => {
+  const classes = useStyles();
+  const {
+    download,
+    label
+  } = props;
+  
+  return (
+    <div>
+      <Grid container spacing={3} direction="row" justify="flex-start" alignItems="center"
```

Comment:
Do we need to use a Grid for a single item ?

Reply:
With a view to upgrade the component in the future, it is more convenient to implement directly a Grid who already possess all the display parameters.